lib/mauth/client/security_token_cacher.rb

```diff
@@ -54,12 +48,17 @@ def security_token_from(response_body)
         def signed_mauth_connection
           require 'faraday'
           require 'mauth/faraday'
```

Comment:
I suppose we can remove the `require 'faraday'` (the faraday-http-cache require at the top of this file would definitely require it internally) and how about moving that `mauth/faraday` require to the top so `require` is not called every single time?

Reply:
done c079247